classes/game.rb

```diff
@@ -1,7 +1,8 @@
-class Game
+class Game < Item
   attr_accessor :title, :multiplayer, :last_played_at
 
   def initialize(title, multiplayer, last_played_at)
+    super()
```

Comment:
The constructor of the Item class expects two arguments.
Currently, trying to create a Game object throws the following error.

![image](https://github.com/henokkhm/microverse-module-4-ruby-group-capstone/assets/110997325/27ad5a3e-db4c-47f2-b780-99024ffae96a)